src/services/shifts.service.ts

```diff
@@ -77,6 +77,15 @@ export class ShiftsService extends Service {
       .then((res) => Promise.resolve(options?.rawResponse ? res : res.status));
   }
 
+  batchDelete(data: { ids: number[] }): Promise<number>;
+  batchDelete(data: { ids: number[] }, options: { rawResponse: true } & Options): Promise<AxiosResponse<any, any>>;
+  batchDelete(data: { ids: number[] }, options: Options): Promise<number>;
+  batchDelete(data: { ids: number[] }, options?: Options) {
```

Comment:
Instead of adding an additional method for multiple deletes, would it be more appropriate to merge this with the delete method above? Maybe as an overload or an updated type like `number | number[]`?
Also don't think we need an object for this parameter - I reckon `ids: number[]` would be more intuitive for a user

Reply:
okay, so i moved the changes on the delete method with type `number | number[]`